metaflow/current.py

```diff
@@ -84,10 +62,6 @@ def get(self, key, default=None):
     def is_running_flow(self):
         return self._is_running
 
-    @property
-    def flow(self) -> FlowSpec:
-        return self._flow
```

Comment:
this is missing

Reply:
I can add this back for sure. Meanwhile I don't see a code usage for this function now.

Reply:
Added back!